fitbenchmarking/core/tests/test_fitting_benchmarking_benchmark_problems.py

```diff
@@ -0,0 +1,133 @@
+"""
+Tests for
+fitbenchmarking.core.fitting_benchmarking.loop_over_benchmark_problems
+"""
+from __future__ import (absolute_import, division, print_function)
+import inspect
+import os
+import unittest
+import mock
+
+from fitbenchmarking import mock_problems
+from fitbenchmarking.utils import fitbm_result
+from fitbenchmarking.core.fitting_benchmarking import \
+    loop_over_benchmark_problems
+from fitbenchmarking.parsing.parser_factory import parse_problem_file
+from fitbenchmarking.utils.options import Options
+from fitbenchmarking.jacobian.SciPyFD_2point_jacobian import ScipyTwoPoint
+
+fitting_dir = "fitbenchmarking.core.fitting_benchmarking"
```

Comment:
This global variable it called different names in the different test files, could a common name be picked such as FITTING_DIR or PARENT_DIR and with comment saying why it is defined

Reply:
Done